src/mame/tvgames/xavix_m.cpp

```diff
@@ -521,6 +521,18 @@ void xavix_i2c_ltv_tam_state::write_io1(uint8_t data, uint8_t direction)
 	}
 }
 
+void xavix_i2c_mj_state::write_io1(uint8_t data, uint8_t direction)
+{
+	if (direction & 0x02)
+	{
+		m_i2cmem->write_sda((data & 0x02) >> 1);
+	}
+
+	if (direction & 0x01)
+	{
+		m_i2cmem->write_scl((data & 0x01) >> 0);
+	}
+}
```

Comment:
You should also release these when the output isn’t being driven, as there will be a pull-up resistor on each of the lines:
```c++
	m_i2cmem->write_sda(BIT(data | ~direction, 1));
	m_i2cmem->write_scl(BIT(data | ~direction, 0));
```